core-bundle/src/InsertTag/ChunkedText.php

```diff
@@ -10,7 +10,7 @@
  * @license LGPL-3.0-or-later
  */
 
-namespace Contao\CoreBundle\Twig\Interop;
+namespace Contao\CoreBundle\InsertTag;
 
 /**
  * @experimental
```

Comment:
Should we remove the `experimental` flag for this class?

Reply:
Done in 0b95a171f81e7aff76dc00b4e42a92d766c0ba87